docs/sources/reference/cli/environment-variables.md

```diff
@@ -35,4 +36,12 @@ Argument                   | Description                                     | D
 ---------------------------|-------------------------------------------------|--------
 `http://proxy.example.com` | The hostname or IP address of the proxy server. |
 
+### GOMEMLIMIT
+
+Normally the [Go runtime][runtime] will release memory back to the Operating System when requested. In certain environments this may cause issues such as Out of Memory errors. The `GOMEMLIMIT` variable is a numeric value in bytes with an optional unit suffix. The supported suffixes include B, KiB, MiB, GiB, and TiB. This should not be treated has a hard limit. The process can use more memory if that memory is required. A rough number is to set `GOMEMLIMIT` to 90% of the maximum amount of memory usage. For example if you wanted to keep memory usage below `10GiB` then use `GOMEMLIMIT=8GiB`.
+
+#### Automatic setting of GOMEMLIMIT
+
+The `GOMEMLIMIT` variable will be automatically set if {{< param "PRODUCT_NAME" >}} can determine the appropriate value. If the `GOMEMLIMIT` value is set before running  {{< param "PRODUCT_NAME" >}} then that setting will be honored. The value will be set to 90% of the cgroup value set. If the limit cannot be determined and no value was explicitly passed then no changes will occur.
```

Comment:
```suggestion
The `GOMEMLIMIT` environment variable is set automatically {{< param "PRODUCT_NAME" >}} can determine the appropriate value.
If you set the `GOMEMLIMIT` environment variable before you run  {{< param "PRODUCT_NAME" >}} then that setting will be used.
The variable value is set to 90% of the `cgroup` value set.
No changes will occur if the limit cannot be determined and you did not explicitly define a  `GOMEMLIMIT` value.
```


I'm not sure what this means: `The variable value is set to 90% of the cgroup value set.`  Are we referring to GOMEMLIMIT as a cvalue? or does this mean that you define GOMEMLIMIT to some value such as `8GiB` and it takes 90% of that? Or... is this the automatic determination part and this sentence belongs to the first sentence?

Reply:
If a cgroup value is available then the GOMEMLIMIT will automatically be set to 90% of the cgroup value.